ztag/stream.py

```diff
@@ -165,6 +166,26 @@ def take(self, pbout):
     def cleanup(self):
         return self.push(noretry=True)
 
+class _KafkaWorkerProcess(multiprocessing.Process):
+
+    def __init__(self, q, topic, main_producer, cert_producer):
```

Comment:
Yes, I confirmed this in the documentation. It's apparently more efficient to pass it between processes than to have multiple instances.